application/controllers/Create_subject.php

```diff
@@ -87,56 +87,55 @@ public function index()
 
         private function _form_view()
         {
-                $inputs['subject_code'] = array(
+                $this->data['subject_code'] = array(

```

Comment:
wag mo na baguhin mga varibles @shikai06 
kaysa sa palitan mo pa sya ng name, para iwas lito sa previous code,

then no need na sya gamitan ng `$this->`  kasi hndi naman sya global.

local variable lang, paki balik na lang ang old var names. then memerge ko na sya